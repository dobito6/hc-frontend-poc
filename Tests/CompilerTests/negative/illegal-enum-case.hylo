//! no-std

struct A {
  case one()
}

enum B {
  case one(x: Void) = ()
}
