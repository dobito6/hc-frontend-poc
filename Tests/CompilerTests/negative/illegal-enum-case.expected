negative/illegal-enum-case.hylo:4.3: error: declaration is not allowed here
  case one()
  ^
negative/illegal-enum-case.hylo:8.21-22: error: enum case with parameters cannot have an explicit definition
  case one(x: Void) = ()
                    ^
